tests/Behat/DrupalContext.php

```diff
@@ -201,4 +202,64 @@ protected function getLanguageIdByName(string $name): string {
     throw new \InvalidArgumentException("Language '{$name}' not found.");
   }
 
+  /**
+   * Redirect user on Node creation page.
+   *
+   * @param string $content_type_name
+   *   Content type name.
+   *
+   * @Given I am visiting the :content_type_name create node page
+   */
+  public function iAmVisitingTheCreateNodePage(string $content_type_name): void {
+    $node_bundle = $this->getEntityTypeByLabel($content_type_name);
+    $this->visitPath('node/add/' . $node_bundle);
+  }
+
+  /**
+   * Check that selectbox with node language selector is hidden.
+   *
+   * @param string $label
+   *   Label of selectbox.
+   *
+   * @Then I should not see selectbox with label :label
+   */
+  public function iShouldNotSeeSelectboxWithLabel(string $label): void {
```

Comment:
The name of the method (and the step) is misleading, we are not checking for selectboxes but rather for a field. We should either rename the function (and the step) or make it so that it only checks for selectboxes